gguf-py/gguf/constants.py

```diff
@@ -312,6 +314,7 @@ class MODEL_TENSOR(IntEnum):
     FFN_GATE_SHEXP       = auto()
     FFN_DOWN_SHEXP       = auto()
     FFN_UP_SHEXP         = auto()
+    FFN_EXPERT_WEIGHTS_B = auto()
```

Comment:
For more consistency in the names, lets change the `EXPERT` to `EXP`. Also, it seems that `PROBS` is better name since this is a bias for the computed expert probabilities: 

```suggestion
    FFN_EXP_PROBS_B      = auto()
```

Reply:
Done